### DISCUSSION TITLE
Directpv cant find drives and saying it is not installed

### DISCUSSION THREAD

**Discussion by Oscardoodle316** [minio/minio#14562] [Q&A]
So i am trying to setup my MinIO Cluster with 4 Rpis and 16 USB drives. The documentation recommends using directpv which I tried but directpv cant find any drives with this command "kubectl directpv drives ls". Also if I try to verify the installation with "kubectl directpv info" it says I have to install directpv which I already did a few steps prior.
How should I proceed further?

**Comment by Oscardoodle316** [comment]
So it seems that directpv works now, somewhat i guess. It only shows me my drives from 2 of my nodes but I am still missing 2 nodes.
Anyone got an idea why this is?

![Unbenannt](https://user-images.githubusercontent.com/79633210/158709513-55891bdf-3cda-479b-b208-e5deb802c1ac.jpg)

**Comment by Oscardoodle316** [comment]
But it still says I have not installed directpv.
![image](https://user-images.githubusercontent.com/79633210/158709899-2104174c-4087-4278-89ca-a214a14a9b2e.png)

**Comment by usersina** [comment]
I have had the same issue but it was a conflict in the plugin version and the manifest version.

In my case, I installed `directpv` in a special way since I had to update the kubelet path using the following two commands
```bash
kubectl krew install directpv
kubectl create -f directpv-install.yaml
```

Although that would break the `info` command since I generated the `directpv-install.yaml` using **a previous version of the plugin**. If that's the case for you, I recommend generating the yaml again and applying that

```
kubectl directpv install -o yaml > directpv-install.yaml
```

**Comment by balamurugana** [comment]
It looks like you are using older version of DirectPV here. It is recommended to use the latest version `v4.0.2`. You could uninstall older DirectPV and try this version.
